crunch/headings-and-subtotals.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Headings and Subtotals"
+audience: all
+---
+
+A common task in the market-research world is to take data that is essentially a scale and collapse it into a smaller number of bins. For example, if you asked people to rate their preference on a scale of 1-10 you might want to see how the people who provide a rating between 1 and 5 compare to those who rated it between 6 and 10 without recoding the data into a new variable. This goes by a number of names like _Top Box_, _Top Two Box_, or _Nets_, but in Crunch we it is called a _Subtotal_. 
+
+Subtotals can be applied to any Categorical variable by opening the variable properties menu and selecting `Headings and Subtotals`. This brings up a list of the variable categories which allows you to insert headings or subtotals in the appropriate location. 
```

Comment:
Maybe make properties be in backticks like `Headings and Subtotals` since it is a similar button. It also would not hurt (thought might be a bit overly pedantic) to have a gif here of someone getting to that menu from the card view.